test/functionalTest/cases/0876_attribute_dates/attribute_dates.test

```diff
@@ -0,0 +1,212 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Attribute dates
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create E1/A1
+# 02. GET E1/A1, see dateCreated == dateModified
+# 03. Update E1/A1
+# 04. GET E1/A1, see dateModified > dateCreated
+#
+
+echo "01. Create E1/A1"
+echo "================"
+payload='{
+  "id": "E1",
+  "A1": {
+    "value": 1
+  }
+}'
+T_0=$(TZ=GMT date +"%Y-%m-%dT%H:%M:%S.00Z")
+orionCurl --url /v2/entities --payload "$payload"
+echo
+echo
+
+
+echo "02. GET E1/A1, see dateCreated == dateModified"
+echo "=============================================="
+orionCurl --url '/v2/entities/E1/attrs/A1?options=dateCreated'
```

Comment:
For metadata, this should be done in a different way. In particular:

```
GET /v2/entities/E1/attrs/A1?metadata=dateCreated
```

Similar to the future way in which this will be done for dateCreated/dateModifiied associated to entities:

```
GET /v2/entities/E1/attrs?attrs=dateCreated
```

Note that the `options=dateCreated` way will be deprecated at the end (although still supported, due to backward compatibility). See https://github.com/telefonicaid/fiware-orion/issues/2455#issuecomment-248883258